packages/core/src/com/communication.ts

```diff
@@ -235,36 +252,47 @@ export class Communication {
     /**
      * Calls a remote method in any opened environment.
      */
-    public callMethod(
-        envId: string,
-        apiId: string,
-        methodName: string,
-        args: SerializableArguments,
-        origin = this.rootEnvId
-    ): Promise<unknown> {
+    public callMethod({
+        envId,
+        api,
+        method,
+        args,
+        origin = this.rootEnvId,
+        serviceComConfig = {}
+    }: ICallMethodOptions): Promise<unknown> {
```

Comment:
This is a performance junction maybe should go back to regular args and also remove default values.